overlays/custom-packages.nix

```diff
@@ -26,6 +26,25 @@
       gala-app = final.callPackage ../user-apps/gala {};
       waypipe-ssh = final.callPackage ../user-apps/waypipe-ssh {};
       wifi-connector = final.callPackage ../user-apps/wifi-connector {};
+
+      labwc = (
+        prev.labwc.overrideAttrs (prevAttrs: {
+          preInstallPhases = ["preInstallPhase"];
+          preInstallPhase = ''
+            substituteInPlace ../docs/autostart \
+             --replace swaybg ${final.swaybg}/bin/swaybg \
+             --replace kanshi ${final.kanshi}/bin/kanshi \
+             --replace waybar ${final.waybar}/bin/waybar \
+             --replace mako ${final.mako}/bin/mako \
+             --replace swayidle ${final.swayidle}/bin/swayidle
+
+             substituteInPlace ../docs/menu.xml \
+             --replace alacritty ${final.foot}/bin/foot
+
+             chmod +x ../docs/autostart
```

Comment:
Please limit chmod to `labwc` user and group only - `chmod ug+x`

Reply:
chmod ug+x actually breaks execution - tested by author

Reply:
As we discussed offline, unfortunately, that is how Nixpkgs suggest to keep runnable scripts.